src/mlagility/version.py

```diff
@@ -1 +1 @@
-__version__ = "3.1.3"
```

Comment:
FYI #328 is making the same change. If #328 merges first, please update this to 3.1.5 here (and I'll do the same in mine if you merge first).

Reply:
Sounds good. I will leave this here unresolved for now to address this if needed when merging.

Reply:
Done! Updated to 3.1.5